lib/node_modules/@stdlib/math/base/special/expm1rel/examples/index.js

```diff
@@ -18,6 +18,13 @@
 
 'use strict';
 
+var randu = require( '@stdlib/random/base/randu' );
 var expm1rel = require( './../lib' );
 
-// TODO: add example code
+var x;
+var i;
+
+for ( i = 0; i < 100; i++ ) {
+	x = (randu()*100.0) - 50.0;
+	console.log( '(e^%d - 1)/%d = %d', x, x, expm1rel( x ) );
```

Comment:
For readability, you may want to truncate the number of decimals which are displayed; e.g., `x.toFixed(3)`.